scapy/contrib/bier.uts

```diff
@@ -0,0 +1,20 @@
+# BIER unit tests
+#
+# Type the following command to launch start the tests:
+# $ test/run_tests -P "load_contrib('bier')"  -P "load_contrib('mpls')"  -t scapy/contrib/bier.uts
+
++ BIER tests
+
+= BIER - build/dissection
+
```

Comment:
You need
`from scapy.contrib.mpls import MPLS`

Our test software only auto-imports the contrib module with the same name as the .uts file